extensions/odoo_theme/static/scss/_font_awesome.scss

```diff
@@ -0,0 +1,947 @@
+/*!
+ *  Based on Font Awesome 4.7.0 by @davegandy - http://fontawesome.io - @fontawesome
+ *  License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)
+ */
+/* FONT PATH
+ * -------------------------- */
+[class^="fa-"] {
+  display: inline-block;
+  font-family: FontAwesome !important;
+  speak: never;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
```

Comment:
I assume this part of the code comes from the `.oi` selector of Oddo UI icons and that you've changed the font-family value? 🙂 
This part isn't necessary, as you've already defined the Font Awesome properties just below (lines 16 to 21).